test/widgets/compose_box_test.dart

```diff
@@ -264,14 +455,15 @@ void main() {
 
     group('attach from media library', () {
       testWidgets('success', (tester) async {
-        final controllerKey = await prepareComposeBox(tester, narrow: ChannelNarrow(eg.stream().streamId));
-        final composeBoxController = controllerKey.currentState!;
+        TypingNotifier.debugEnable = false;
+        addTearDown(TypingNotifier.debugReset);
 
+        final controllerKey = await prepareComposeBox(tester,
+          narrow: ChannelNarrow(eg.stream().streamId),
+          topic: 'some topic', content: 'see image: ');
+        final composeBoxController = controllerKey.currentState!;
         // (When we check that the send button looks disabled, it should be because
         // the file is uploading, not a pre-existing reason.)
-        composeBoxController.topicController!.value = const TextEditingValue(text: 'some topic');
-        composeBoxController.contentController.value = const TextEditingValue(text: 'see image: ');
-        await tester.pump();
```

Comment:
This comment no longer makes sense — it was here to explain why we're setting the topic and content.